### PR TITLE
5xx error alerts

### PR DESCRIPTION
**Pull request by pluckyswan**
# 5xx error alerts

Added terraform for 5xx error Azure alerts. Because it was deemed as higher priority than 4xx errors, the threshold is lower and more frequent. It was tested in internal by creating and hitting a temporary bad health check that returns a 500. Alert appeared in Slack when activated and deactivated. 

## Issue

https://github.com/CDCgov/trusted-intermediary/issues/1394

## Checklist

**Note**: You may remove items that are not applicable

### CODE REVIEW

File: operations/template/alert.tf
Comment (halprin) [inline operations/template/alert.tf:130]: This makes sense to have a `threshold` of `1` and checking every minute.  And sure, why not `5` minutes ago even though we only last checked 1 minute ago.

**Review by halprin — COMMENTED** [review]
I see these PRs build on each other.  I'd apply the feedback about the `tags` in the previous PR to this PR too.

File: operations/template/alert.tf
Comment (pluckyswan) [inline operations/template/alert.tf:134]: I don't think this should work since it's not a metric option but I'm unsure how to verify. Maybe use HttpServerErrors or a log alert instead if it doesn't.

Comment (halprin) [reply operations/template/alert.tf:134]: Hmmm, we could inject a crash into our code so we have a guaranteed 500.  Then deploy to the internal environment.

**Comment by halprin** [comment]
Since this PR seems to have been branched off of our 4xx alert branch, I would wait until #1465 gets merged before this is merged.

File: operations/template/alert.tf
Comment (halprin) [inline operations/template/alert.tf:172]:
```suggestion
  window_size         = "PT5M" // Every Check looks back 5 min for 5xx errors
```

Comment (halprin) [reply operations/template/alert.tf:172]: Besides this one thing, this alert looks good.

**Review by halprin — APPROVED** [review]
Kick ass!

**Pull request merged**
